api/tests/test_game_state_server.py

```diff
@@ -1,40 +1,85 @@
-import game_state_server as gss
+import pytest
 
+from game_state_server import GameStateServer, MessageError, Token
+from room_store import MemoryRoomStore
 
-test_token = {
-    'id': 'valid-id',
-    'icon_id': 'valid_icon_id',
+
+TEST_ROOM_ID = 'test_room'
+TEST_CLIENT = 'test_client'
+valid_data = {
+    'id': 'some_id',
+    'icon_id': 'some_icon_id',
     'start_x': 0,
     'start_y': 0,
     'start_z': 0,
     'end_x': 1,
     'end_y': 1,
     'end_z': 1,
 }
+valid_update = {'action': 'create', 'data': valid_data}
+
+
+@pytest.fixture
+def set_up_gss():
```

Comment:
Looks like fixtures follow the name format of lower_case_noun (i.e. just gss for this one) in this example: https://docs.pytest.org/en/latest/fixture.html Is there a reason you chose the set_up prefix?

Reply:
Nope, will change.